eng/Directory.Build.Data.targets

```diff
@@ -43,6 +44,8 @@
   <!-- Add Package Icon to DataPlane Packages -->
   <ItemGroup>
     <None Include="$(RepoEngPath)/images/pkgicon.png" Pack="true" PackagePath=""/>
+    <None Condition="Exists('$(MSBuildProjectDirectory)/../CHANGELOG.md')" Include="$(MSBuildProjectDirectory)/../CHANGELOG.md" Pack="true" PackagePath=""/>
+    <None Condition="Exists('$(MSBuildProjectDirectory)/../README.md')" Include="$(MSBuildProjectDirectory)/../README.md" Pack="true" PackagePath=""/>
```

Comment:
One other thing we need to do is start to run the `eng/pipelines/templates/scripts/replace-relative-links.yml@azure-sdk-tools` step before we produce the packages so that the links in the packages are stable. We probably want to do that for readmes and changelogs. Lets not block this PR on this but can you please file another issue to track that work.

Reply:
Issue filled here https://github.com/Azure/azure-sdk-for-net/issues/9210